controller/api/views.py

```diff
@@ -2,176 +2,117 @@
 RESTful view classes for presenting Deis API objects.
 """
 
-from __future__ import absolute_import
-from __future__ import unicode_literals
-
 from django.conf import settings
-from django.contrib.auth.models import AnonymousUser, User
 from django.core.exceptions import ValidationError
-from django.utils import timezone
-from guardian.shortcuts import assign_perm
```

Comment:
as you mentioned, if it's all from the same root package then the comma-separated `import` is PEP8 compliant. It's just the root imports that need separate line statements :)